### ISSUE TITLE
TP Template does not build on Mono

### ISSUE THREAD

**Issue by sergey-tihon** [fsprojects/FSharp.TypeProviders.SDK#265]
### Description

TP created from the template does not build on `macOS`/`mono`.

### Repro steps

Please provide the steps required to reproduce the problem

1. `dotnet new -i FSharp.TypeProviders.Templates`
1. `dotnet new typeprovider -n LemonadeProvider -lang F#`
1. `cd LemonadeProvider`
1. `mono .paket/paket.exe update`
1. `dotnet build -c release`

### Expected behavior

Project should compile

### Actual behavior

<img width="1210" alt="lemonadeprovider_runtime_fsproj_ _untitled__workspace_" src="https://user-images.githubusercontent.com/1197905/46243581-9ed95780-c3de-11e8-98de-f3e03f9581da.png">

### Analysis

This is looks very strange for me, because `fsharp.typeproviders.templates.5.0.0.nupkg` contains `netfx.props` file and `fsproj` files import it.
<img width="1251" alt="win10_-_1803__april_" src="https://user-images.githubusercontent.com/1197905/46243629-1909dc00-c3df-11e8-841d-41fc00584d9c.png">

but `dotnet new typeprovider -n LemonadeProvider -lang F#` creates a folder without `netfx.props` and without imports.
<img width="1510" alt="lemonadeprovider_tests_fsproj_ _untitled__workspace_" src="https://user-images.githubusercontent.com/1197905/46243649-638b5880-c3df-11e8-97b6-abcbab5cdbc1.png">

### Related information 

* Operating system = `macOS Mojave (10.14)` 
* `.NET Core 2.1.401`, `Mono 5.12.0.301`

**Comment by sergey-tihon** [comment]
@sgtz proposed an idea for workaround, that we should not only import props file 
```xml
<Import Project="..\..\netfx.props" />
```
but also add it as a resource 
```xml
<ItemGroup>
    <None Include="..\..\netfx.props" />
    ...
</ItemGroup>
```

as a result, `MyProvider.Tests.fsproj`will look like this
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <Import Project="..\..\netfx.props" />
  <PropertyGroup>
    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
  </PropertyGroup>
  <ItemGroup>
    <None Project="..\..\netfx.props" />
    <None Include="app.config" />
    <Compile Include="MyProvider.Tests.fs" />
    <None Include="paket.references" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0"/>
    <ProjectReference Include="..\..\src\MyProvider.Runtime\MyProvider.Runtime.fsproj">
      <Name>MyProvider.Runtime</Name>
      <Project>{7e90d6ce-a10b-4858-a5bc-41df7250cbca}</Project>
      <Private>True</Private>
    </ProjectReference>
  </ItemGroup>
  <Import Project="..\..\.paket\Paket.Restore.targets" />
</Project>
```

**Comment by sergey-tihon** [comment]
Looks like my mistake, I downloaded latest version (5.0.0) from NuGet on Windows manually and it does not containt `netfx.props`. This means that we need to release new version of template...

<img width="1224" alt="win10_-_1803__april_" src="https://user-images.githubusercontent.com/1197905/46245036-85410b80-c3f0-11e8-8127-2de4943b21b2.png">

**Comment by sergey-tihon** [comment]
Confirm that it works OK with NuGet package created manually from `master` `/templates` folder.

**Comment by sergey-tihon** [comment]
Fixed in [v5.1.0](https://www.nuget.org/packages/FSharp.TypeProviders.Templates/5.1.0)

**Comment by dsyme** [comment]
Thank you @sergey-tihon !
